src/test_application/OSPMagnum.h

```diff
@@ -91,3 +93,5 @@ class OSPMagnum : public Magnum::Platform::Application
 void config_controls(OSPMagnum& rOspApp);
 
 }
+
+std::vector<osp::ButtonVarConfig> parse_control(std::string_view str);
```

Comment:
I recommend putting some doxygen style documentation in the header files for functions declarations.

Reply:
Additionally, it looks like this functions isn't used by this file.

Would it make sense for it to have it's own header file instead?